app/assets/javascripts/spree/backend/edit_inline_locales.js.coffee.erb

```diff
@@ -0,0 +1,56 @@
+Spree.EditInlineLocales = Backbone.View.extend(
+  initialize: ->
+    @editing = false
+    @render()
+
+  events:
+    'click [data-action=edit]': 'onEdit'
+    'click [data-action=save]': 'onSave'
+    'click [data-action=cancel]': 'onCancel'
+
+  onEdit: (e) ->
+    return if @$el.hasClass('editing')
```

Comment:
I think we can use the `editing` value for the conditional, that way we can stop adding and removing class to the element